nkms/policy/models.py

```diff
@@ -288,3 +288,36 @@ def __eq__(self, other):
 
     def __iter__(self):
         return iter(self.ids)
+
+
+class WorkOrder(object):
+    def __init__(self, kfrag_hrac, pfrags, receipt_bytes, receipt_signature, bob_pubkey_sig, ursula_id=None):
+        self.kfrag_hrac = kfrag_hrac
+        self.pfrags = pfrags
```

Comment:
Why do we have many `pfrags`? Did you mean to be able to process many ciphertexts in one order?
The input (which we call `pfrag`) is actually the same for all Ursulas, so it's not really correct to call it `frag`